.github/workflows/main.yml

```diff
@@ -34,13 +34,16 @@ jobs:
         sudo apt-get install ccache
         sudo apt-get install lld
  
+    # install clang for LLVM build
+    - name: install clang
+      run: sudo apt-get install -y clang
+
     # setup LLVM
     - name: install a specific version of LLVM
       working-directory: ${{github.workspace}}
       run: |
-        git clone https://github.com/llvm/llvm-project.git
+        git clone --depth 1 --branch release/19.x https://github.com/llvm/llvm-project.git
```

Comment:
So this 19.x could work same as the commit `cd70802`? No error during our dialect's building?